elide-datastore/elide-datastore-inmemorydb/src/main/java/com/yahoo/elide/datastores/inmemory/InMemoryTransaction.java

```diff
@@ -57,7 +58,8 @@ public void save(Object object) {
         }
         String id = dictionary.getId(object);
         if (id.equals("0")) {
-            setId(object, dictionary.getId(createObject(object.getClass())));
+            createObject(object, null);
```

Comment:
Let's see if this is still needed.  Let's throw an exception if id ever equals 0.